### PR TITLE
Create a manual python file to analyse merger_pitch_FR status

### PR DESCRIPTION
**Pull request by Jibec**
I want to improve the [analyser_merge_pitch_FR.py](https://github.com/osm-fr/osmose-backend/blob/master/analysers/analyser_merge_pitch_FR.py).

Let's start by a status!
As I see this merge analyser is old, I feel like storing such scripts may help future contributors.
Let's discuss if the is the correct place to to this, and if the method is fine for you.

Here is the output of today's run:
```
2023-03-08 23:44:13,166 - Jetzt geht's los
2023-03-08 23:44:13,166 - Loading source files
2023-03-08 23:44:16,303 - Step 1: searching for missing merge values
2023-03-08 23:44:16,338 -   Step 1.1: do we have all 'Type d'équipement sportif' in '../merge_data/pitch_FR.mapping.csv'?
2023-03-08 23:44:16,338 -     'Salle multisports (gymnase)' does not exist in '../merge_data/pitch_FR.mapping.csv', it is 18184 equipments
2023-03-08 23:44:16,340 -     'Parcours d'obstacles' does not exist in '../merge_data/pitch_FR.mapping.csv', it is 434 equipments
2023-03-08 23:44:16,340 -     'Courses sur piste' does not exist in '../merge_data/pitch_FR.mapping.csv', it is 192 equipments
2023-03-08 23:44:16,340 -     'Salle de volley-ball' does not exist in '../merge_data/pitch_FR.mapping.csv', it is 65 equipments
2023-03-08 23:44:16,340 -     'Terrain de padel' does not exist in '../merge_data/pitch_FR.mapping.csv', it is 158 equipments
2023-03-08 23:44:16,341 -     'Terrain de polo-vélo' does not exist in '../merge_data/pitch_FR.mapping.csv', it is 3 equipments
2023-03-08 23:44:16,341 -     'Anneau de vitesse' does not exist in '../merge_data/pitch_FR.mapping.csv', it is 1 equipments
2023-03-08 23:44:16,371 -   -> current coverage is 93.87%, (missing: 19037)
2023-03-08 23:44:16,371 -   Step 1.2: do we have all values from '../merge_data/pitch_FR.mapping.csv' in 'Type d'équipement sportif' column?
2023-03-08 23:44:16,371 -     'Site artificiel' does not exist in 'data-es.csv'
2023-03-08 23:44:16,371 -     '' does not exist in 'data-es.csv'
2023-03-08 23:44:16,371 -     'Course sur piste' does not exist in 'data-es.csv'
2023-03-08 23:44:16,372 -     'Parcours d'obstacle' does not exist in 'data-es.csv'
2023-03-08 23:44:16,372 -     'Salle multisports' does not exist in 'data-es.csv'
2023-03-08 23:44:16,372 -     'Salle de volley ball' does not exist in 'data-es.csv'
2023-03-08 23:44:16,372 -     'Terrain de paddle' does not exist in 'data-es.csv'
2023-03-08 23:44:16,373 - Step 2: is our surface coverage exhaustive?
2023-03-08 23:44:16,409 -   Step 2.1: do we have all 'Nature du sol' in '../merge_data/pitch_FR.mapping.csv'?
2023-03-08 23:44:16,409 -     'Surface naturelle' does not exist in '../merge_data/pitch_FR.mapping.csv', it is 44807 equipments
2023-03-08 23:44:16,409 -     'Carrelage' does not exist in '../merge_data/pitch_FR.mapping.csv', it is 12900 equipments
2023-03-08 23:44:16,409 -     'Stabilisé/cendrée' does not exist in '../merge_data/pitch_FR.mapping.csv', it is 24302 equipments
2023-03-08 23:44:16,409 -     'Synthétique (hors gazon)' does not exist in '../merge_data/pitch_FR.mapping.csv', it is 50813 equipments
2023-03-08 23:44:16,409 -     'Parquet' does not exist in '../merge_data/pitch_FR.mapping.csv', it is 16799 equipments
2023-03-08 23:44:16,409 -     '' does not exist in '../merge_data/pitch_FR.mapping.csv', it is 2869 equipments
2023-03-08 23:44:16,409 -     'Sciure/copeaux' does not exist in '../merge_data/pitch_FR.mapping.csv', it is 325 equipments
2023-03-08 23:44:16,409 -     'Autre' does not exist in '../merge_data/pitch_FR.mapping.csv', it is 158 equipments
2023-03-08 23:44:16,409 -     'Glace' does not exist in '../merge_data/pitch_FR.mapping.csv', it is 318 equipments
2023-03-08 23:44:16,436 -   -> current coverage is 50.61%, (missing: 153291)
2023-03-08 23:44:16,436 -   Step 2.2: do we have all values from '../merge_data/pitch_FR.mapping.csv' in 'Nature du sol' column?
2023-03-08 23:44:16,436 - Färtig !
```

### CODE REVIEW

**Comment by frodrigo** [comment]
At least at the time it was created. A full mapping was not possible. I guess it is always the case.

A Part of your check could be done in unitary tests.

**Comment by Jibec** [comment]
I'm done with this review. Indeed, many items needs the user to review manually the sport to better define it.
We could probably do much better by guessing from name and column "Type de particularité de l'installation", but this goes beyond my technical skills.

One single tag invention I did is sport=**model_sailing** by following the **model_aerodrome** and **model_car_racing**

I have no idea on how to do this with unit test, as far as I know, in my job we use unit test is done to test the software behavior only and not the data.

**Comment by Jibec** [comment]
@frodrigo : I don't know how to fix the mypy CI issue, should I keep it simple and move the dictionary back to the main file or should I improve the way I share this mapping file? Any advice?

**Comment by frodrigo** [comment]
> @frodrigo : I don't know how to fix the mypy CI issue, should I keep it simple and move the dictionary back to the main file or should I improve the way I share this mapping file? Any advice?

`dictionaries` is not for python file, make a csv file, or keep it in code. But you file is not really a dictionary (is a mapping). I think it is best to keep in code.

**Comment by Jibec** [comment]
ok, I moved it back to the code, let's hope it's fine now.

File: tools/manual_review_merge_pitch_FR.py [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    "Métal": "metal",
+    "Stabilisé/cendrée": "compacted",
+    "Synthétique (hors gazon)": "tartan",
+    "Sciure/copeaux": "woodchips"
+    # 12900 Carrelage
+    # 16799 Parquet
+}
+
+
+log.info("Jetzt geht's los")
```
Comment (frodrigo) [inline tools/manual_review_merge_pitch_FR.py:44]: Please, use English (there is other)

File: tools/manual_review_merge_pitch_FR.py [hunk 2]
```diff
@@ -0,0 +1,102 @@
+"""
+ This file helps to review the data used to merge information
+ 1. download file from https://equipements-sgsocialgouv.opendatasoft.com/explore/dataset/data-es/information/
+ 2. store it ad data-es.csv in the current folder
```
Comment (Famlam) [inline tools/manual_review_merge_pitch_FR.py:4]: Typo: store it *as*

**Comment by frodrigo** [comment]
Thank you. Merged. But not deployed yet.

**Pull request merged**